phpspec.yml.dist

```diff
@@ -0,0 +1,4 @@
+suites:
```

Comment:
nevím zda to tu nechávat, když to nepoužíváme?

Reply:
> je to si ce jednorázovka, ale použiješ to při dalším upgradu?

JJ, jen vyměním `LevelSetList::UP_TO_PHP_80` za vyšší PHP.

Reply:
> nevím zda to tu nechávat, když to nepoužíváme?

Nikdy jsem přes PHPSpec nic negeneroval, ale dle dokumentace to máme správně a nechal bych to tak, ať máme config pro phpspec kompletní http://phpspec.net/en/stable/cookbook/configuration.html

Reply:
a k čemu to tu bude, když to nebudeme používat?

Reply:
PHPSpec nepoužíváme, takže config byl k ničemu.

Smazal jsem ho.